dapp_runner/descriptor/dapp.py

```diff
@@ -46,10 +48,15 @@ def __ports_factory(value: str) -> PortMapping:  # type: ignore [misc]  # noqa
 class ServiceDescriptor(BaseDescriptor["ServiceDescriptor"]):
     """Yapapi Service descriptor."""
 
+    def __ip_list() -> List[str]:  # type: ignore [misc]  # noqa
+        return []
+
     payload: str
     entrypoint: List[List[str]]
     network: Optional[str] = None
+    ip: Optional[List[str]] = field(default_factory=__ip_list)
```

Comment:
```suggestion
    ip: Optional[List[str]] = field(default_factory=list)
```

I'm pretty sure this should work.
If it does, remember to remove `__ip_list` function.

Reply:
no, mypy complains...

Reply:
okay, extracted the function to the outer scope so it doesn't trigger any mypy warnings

Reply:
... and removed altogether in favor of a simple `# type: ignore`